packages/hoppscotch-backend/src/user-history/user-history.service.ts

```diff
@@ -188,6 +188,16 @@ export class UserHistoryService {
     return E.right(deletionInfo);
   }
 
+  /**
+   * Delete all user history from DB
+   * @returns a boolean
+   */
+  async deleteAllHistories() {
+    await this.prisma.userHistory.deleteMany();
```

Comment:
this should be inside a try catch block, what if there is an issue with database connection ?